web/src/components/Result.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div class="col text-center">
+    <h4>Your indexed rent amounts to €{{ result.new_rent.toFixed(2) }}</h4>
```

Comment:
consider using a filter
```suggestion
    <h4>Your indexed rent amounts to {{ result.new_rent | currency }}</h4>
```
where 
```js 
  filters: {
    currency (val) {
      return +val.toLocaleString('en-US' , { 
        minimumFractionDigits: 2, 
        maximumFractionDigits: 2, 
        currency: 'EUR', 
        style: 'currency' 
      })
    }
  }
```